src/test/scala/it/pagopa/interop/notifier/model/NotificationObjectTypeSpec.scala

```diff
@@ -0,0 +1,25 @@
+package it.pagopa.interop.notifier.model
+
+import cats.syntax.all._
+import it.pagopa.interop.notifier.model.NotificationObjectType._
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpecLike
+import org.scanamo.DynamoValue
+
+class NotificationObjectTypeSpec extends AnyWordSpecLike with Matchers {
+
+  "NotificationObjectType values" should {
+    "be correctly deserialized for all object types" in {
+
+      import scala.reflect.runtime.{universe => ru}
```

Comment:
I know, sorry.
Feel free to suggest any other solution to avoid forgetting to update the codec